app/controllers/providers/application_merits_task/opponents_mental_capacities_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Providers
+  module ApplicationMeritsTask
+    class OpponentsMentalCapacitiesController < ProviderBaseController
+      def show
+        @form = Opponents::MentalCapacityForm.new(model: opponent)
+      end
+
+      def update
+        @form = Opponents::MentalCapacityForm.new(form_params)
+        render :show unless update_task_save_continue_or_draft(:application, :opponent_mental_capacity)
+      end
+
+    private
+
+      def opponent
+        @opponent ||= legal_aid_application.opponent || legal_aid_application.build_opponent
```

Comment:
more of a general question and not a blocker but should we be erroring in situations where the opponent does not exist?